R/query_dataset.R

```diff
@@ -0,0 +1,57 @@
+#' Query a data set
+#'
+#' @description
+#' A short description...
```

Comment:
Short indeed... Going to expand here or when doing the POST code?

Reply:
Have added some initial text now.